pkgs/tools/X11/sbs/default.nix

```
@@ -0,0 +1,29 @@
+{ lib, stdenv, fetchzip, libX11, imlib2, libXinerama, pkg-config }:
+
+stdenv.mkDerivation rec {
+  pname = "sbs";
+  version = "1.0.0";
+
+  src = fetchzip {
+    url = "https://github.com/onur-ozkan/${pname}/archive/refs/tags/v${version}.tar.gz";
+    sha256 = "sha256-Zgu9W/3LwHF/fyaPlxmV/2LdxilO1tU0JY/esLnJVGY=";
+  };
```

```suggestion
  src = fetchFromGitHub {
    owner = "onur-ozkan";
    repo = "sbs";
    rev = "${version}"
    sha256 = "sha256-Zgu9W/3LwHF/fyaPlxmV/2LdxilO1tU0JY/esLnJVGY=";
  };
```
`fetchFromGitHub` is better here. It has some Github related optimizations. The hash will probably also change.